lib/clerk/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Clerk
-  VERSION = "3.0.0"
+  VERSION = "4.0.0.beta1"
```

Comment:
I kept the same naming with the ones in the past. We are using this for some time 😓 . I will update it and let's try to follow semver forma from now on.
Also i think that the correct is `4.0.0-beta.1`

Reply:
You're right, I made a typo above. :facepalm: 

The correct value is `4.0.0-beta.1`.

Reply:
Using the `4.0.0-beta.1` in `version.rb` produces a pre-release tag (which i expect is what we want) `4.0.0.pre.beta1`. Should i refer to `4.0.0-beta1` or `4.0.0.pre.beta1` in `CHANGELOG.md`?
I also using the `4.0.0-beta.1` in version file would probably result in making everyone that wants to use the beta having to install `v4.0.0.pre.beta1` instead of `4.0.0.beta1`.
Is this okay?

Reply:
After some discussion with @gkats we checked what is the current ruby way (checked top downloaded gems) and concluded that we will keep the `4.0.0.beta1` format instead for consistency with the other gems and our previous gem versions.